recipes/tibi/build_tibi.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CPPFLAGS="${CPPFLAGS} -D_NODEBUG -D_OMP -D_ELPA"
+FFLAGS="${FFLAGS} -I${PREFIX}/include -I${PREFIX}/include/elpa_onenode_openmp/modules -fopenmp -funroll-loops"
+LDFLAGS="${LDFLAGS} -lgomp -lxsmmf -lxsmm -lxsmmext -lelpa_onenode_openmp -lblas -llapack"
+make clean
+make -j${CPU_COUNT}
```

Comment:
Is there a make check we can run here?

Reply:
If with this you refer to a testsuite for the `tibi` package, then that is happening in the `test` section for `tibi-python`. That testsuite requires the Python interface (supplied by the `tibi-python` package) and serves to check both the produced executables (supplied by the `tibi` package) and the Python interface itself.

Reply:
> If with this you refer to a testsuite for the `tibi` package, then that is happening in the `test` section for `tibi-python`. That testsuite requires the Python interface (supplied by the `tibi-python` package) and serves to check both the produced executables (supplied by the `tibi` package) and the Python interface itself.

I saw that one. Just wanted to know if there was a also a `make check`, no problem if there isn't but if it exist, we should run it here.